src/main/resources/narayana-checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+
+    <property name="cacheFile" value=""/>
```

Comment:
I think we could try to define the `cacheFile` as `target/checkstyle.cache`  to not use the property but still considering that. Or maybe if it's not a good idea from your perspective to have it here then I think let's completely remove this configuration from here (by default it's `null`). We can define that in the `pom.xml` in Narayana project then with property `checkstyle.cache.file`.